src/components/ExecuteButton.js

```diff
@@ -45,12 +37,9 @@ const ExecuteButton = ({
     }
   }, [remediationStatus]);
 
-  const buttonWithTooltip = (isDisabled) => {
+  const buttonWithTooltip = () => {
```

Comment:
Why are you deleting `isDisabled` parameter?

Reply:
I can access it from props, so there is no need to pass it as a param, or not?

Reply:
Oh missed the isDisabled param passing in function call, just fixed, thanks